### PR TITLE
🌱 (cleanup/e2e tests): cert-manager and prometheus installs into e2e tests

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description

🌱 (cleanup/e2e tests): cert-manager and prometheus installs into e2e tests

## Motivation
Maintainability

### CODE REVIEW

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

File: test/e2e/utils/test_context.go [hunk 1]
```diff
@@ -29,6 +29,14 @@ import (
 	. "github.com/onsi/ginkgo/v2" //nolint:golint,revive
 )
 
+const (
+	certmanagerVersion        = "v1.9.1"
```
Comment (camilamacedo86) [inline test/e2e/utils/test_context.go:33]:
```suggestion
	certmanagerVersion        = "v1.5.3"
```

**Pull request merged**